docs/design_decisions/DR-004-infra.rst

```diff
@@ -0,0 +1,134 @@
+..
+   Copyright (c) 2025 Contributors to the Eclipse Foundation
+
+   See the NOTICE file(s) distributed with this work for additional
+   information regarding copyright ownership.
+
+   This program and the accompanying materials are made available under the
+   terms of the Apache License Version 2.0 which is available at
+   https://www.apache.org/licenses/LICENSE-2.0
+
+   SPDX-License-Identifier: Apache-2.0
+
+DR-004-Infra: Bi-directional traceability in docs
+=================================================
+
+.. dec_rec:: Bi-directional traceability in docs
+   :id: dec_rec__infra__traceable_docs
+   :status: accepted
+   :context: Infrastructure
+   :decision: Single documentation build
+
+   Date: 2025-10-24
+
+Context / Problem
+-----------------
+
+The S-CORE project adopted a multi-repo strategy.
+Since `S-CORE aims for compliance with regulations like ASPICE <https://eclipse-score.github.io/process_description/main/standards/index.html#standards>`_,
+bi-directional traceability is necessary (see ``gd_req__req_traceability``) and should be made convenient.
+How to provide this in S-CORE documentation?
+
+Previously, there was this unfinished
+`Decision Record: Multi-Repository Documentation Setup in SCORE <https://github.com/AlexanderLanin/score/blob/docs-multirepo/docs/_tooling/decision-records/multirepo.md>`_.
+
+For discussion, this subset of S-CORE modules shall serve as an example:
+
+.. image:: img/DR-004-example.png
+
+
+Goals and Requirements
+^^^^^^^^^^^^^^^^^^^^^^
+
+1. We want to enable each module to work independently and efficiently, i.e. build their own documentation quickly.
+   (Sphinx builds are annoyingly slow in any case but that is not the core point of this decision record.)
+2. We want a complete and consistent documention for a release, i.e. everything with backlinks.
+   Regulations only require that *eventually* though and during development slight inconsistencies are acceptable.
```

Comment:
Inconsistencies will lead to build failures during development. I would address this more to convinience: we do not have a hosted html version of a every specific dependency/version.